rosidl_typesupport_c/bin/rosidl_typesupport_c

```diff
@@ -22,14 +21,7 @@ def main(argv=sys.argv[1:]):
         help='The typesupports to be used')
     args = parser.parse_args(argv)
 
-    try:
-        return generate_c(
-            args.generator_arguments_file,
-            args.typesupports,
-        )
-    except UnknownMessageType as e:
-        print(str(e), file=sys.stderr)
-        return 1
+    return generate_c(args.generator_arguments_file, args.typesupports)
```

Comment:
`generate_c` doesn't return a value:

https://github.com/ros2/rosidl_typesupport/blob/f37ef148e5a1599aea59ff5390350965931f6152/rosidl_typesupport_c/rosidl_typesupport_c/__init__.py#L18-L24

Maybe it should be updated to return the result of `generate_files`.

Same for `generate_cpp`

Reply:
It didn't before and doesn't have to. The implicit return value is `None`. The return value is used for `sys.exit()`.

Reply:
:ok_hand: